linode/manager
# [SDK] Change: Purify NodeBalancer Logic

🚨 This NodeBalancer code needs to be completely re-written. I discovered a handful of bugs already existing in production, and I'm definitely going to make this a priority next tdt. When reviewing, please keep this in mind. 🚨 

Also please look at comments below vvvvv

## Description

Gets rid of all Cloud Manager specific requirements from the NodeBalancer SDK code. This includes types, client-side validation, and error handling.

## Type of Change
- Non breaking change ('update', 'change')

## To Test

Try Creating/Editing NodeBalancer Configs and Nodes and make sure you're successfully able to submit the form and getting correct error messaging.
